### DISCUSSION TITLE
Support Wechat Pay in hyperswitch

### DISCUSSION THREAD

**Discussion by KevinSheeranxyj** [juspay/hyperswitch#6618] [Ideas / Feature Requests]
Hi everyone, 

I'm a rust developer and I'm writing to ask wether I'm allowed to contribute my code with Wechat pay integration in hyperswitch.

Best regards, 
Kevin Sheeran

**Comment by deepanshu-iiitu** [comment]
Hi @KevinSheeranxyj 
Yes, you're welcome to contribute to Hyperswitch! Have you started working on the WeChat Pay integration in Hyperswitch? If so, could you please share the details about which payment processor you've chosen and the specific form of WeChat Pay integration you're implementing?

**Comment by KevinSheeranxyj** [comment]
yes, I'm implementing the WeChat Pay integration with Rust, as I noticed that Stripe supported WeChat Pay, so I'm glad they have offered some methods, such as get transaction fee, supported currencies, settlement time. And WeChat Pay supported various intrgration methods, including QR code payment, in-app payment, and web payment.

**Comment by ApiZimbah** [comment]
Than why I am still obligated to pay with Mastercard instead?

**Comment by videni** [comment]
Hi, @KevinSheeranxyj , did you finish the integration please?
